# `pureseqtmr_run`

Branch   |[![Travis CI logo](man/figures/TravisCI.png)](https://travis-ci.org)
---------|--------------------------------------------------------------------------------------------------------------------------------------------
`master` |[![Build Status](https://travis-ci.org/richelbilderbeek/pureseqtmr_run.svg?branch=master)](https://travis-ci.org/richelbilderbeek/pureseqtmr_run) 
`develop`|[![Build Status](https://travis-ci.org/richelbilderbeek/pureseqtmr_run.svg?branch=develop)](https://travis-ci.org/richelbilderbeek/pureseqtmr_run)

Determine the topology of a membrane protein using [pureseqtmr](https://github.com/richelbilderbeek/pureseqtmr).

## What is `pureseqtmr_run`?

`pureseqtmr_run` allows you to determine the topology of a membrane protein,
in an easy-to-use way.

To do so, `pureseqtmr_run` uses the infrastructure of GitHub and Travis CI. 
For simple proteomes, this costs nothing. Because everything happens online, 
no installation is needed :+1:.

## How to use `pureseqtmr_run` online?

  1. Get a GitHub account
  2. Get a Travis CI account (one can simply log in with GitHub)
  3. Fork this repo
  4. Activate Travis CI for your fork
  5. Change the alignment FASTA file, called `my_proteome.fas`
  6. See the results in the Travis CI build log

